src/utils/common.ts

```diff
@@ -82,3 +76,12 @@ export const sumAssetsBalanceWorth = (supportedAssets: IAssetWithBalance[]) => {
     return sum;
   }, 0);
 };
+
+export const formUrlOptions = (options: { [key: string]: string }): string => {
```

Comment:
I'd say rename it to `create` or `build` instead of `form`, I read it as `from` and only when I got to the end I realized it's not likely `from`

Reply:
yeah i agree that sounds better